python/README.md

```diff
@@ -1,5 +1,158 @@
+
+- dj-database-url==0.4.1, Django==1.9.6, django-rest==3.3.3, django-widget-tweaks==1.4.1, gunicorn==19.5.0, psycopg2==2.6.1, whitenoise==3.0
+
+### Vagrant Base-box
+
+Ubuntu 14.04 LTS based box minimal/trusty64, hosted at https://atlas.hashicorp.com/minimal/boxes/trusty64
+
+## Usage
+
+Follow these tutorials to see how you can use Zeus for Python and Django stack:
+
+[asciinema Tutorial by byteknacker](https://asciinema.org/a/1u9zm99yzpz6v1b95wv6mrppn)
+
+or 
+
+[Video Tutorial by byteknacker](https://youtu.be/RbQ1qtOVSJc)
+
+Follow these instructions precisely to get started with Zeus:
+
+1. Navigate to the directory inside which you wish to create the new directory for your project.
+2. ```git clone https://github.com/alayek/zeus```
+3. ```chmod 755 zeus/zeus```
+4. ```zeus/zeus yournewprojectdir```
+5. Wait until the provision is done for your new Vagrant environment. Take a nap or get a cup of coffee.
+6. Follow the input prompt to setup your GitHub and git credentials.
+7. Start coding right away inside the synced folder of the Vagrant VM that is synced with your local directory of yourprojectdir.
+8. When you finished coding, and you wish to push to Heroku, simply follow the instruction on (Heroku Documentation)[https://devcenter.heroku.com/categories/python]
+9. ```git push heroku master```
+10. Hurray! You got your first Django app done with Zeus! Please give us some feedback by visiting our (Gitter Chat)[https://gitter.im/FreeCodeCamp/vagrant].
+
+If you get stuck, just visit our Gitter Chat, and you get help asap from collaborators!
+
+### Test Your Setup
+
+Optionally, you might wish to test the development environment setup by following these steps:
+
+- Test manually by checking whether all the features we promised are installed inside your Vagrant VM.
+- Use `date && vagrant up > logfile.log && date` instead of just `vagrant up` to log all printouts of `vagrant up` into the `logfile.log` with start time and end time of the execution. (The terminal prints nothing during the entire setup, please don't panic, everything is written into the `logfile.log`). Finally, inspect `logfile.log` to ensure no installation error occurred.
+- Run vagrant halt and restart with `vagrant up` and `vagrant ssh`. Ensure the state of the VM is the same as before, run another test if necessary.
+- Automated testing:
+
+It is assumed that these tests would be run inside the standard virtualenv of the `vagrant ssh` session.
+
+It would require the `py.test` module to run it, which can be installed as:
+
+```bash
+$ pip install pytest
+```
+
+Then navigate to the top-level directory of the Vagrant synced folder:
+
+```bash
+$ cd /vagrant
+```
+
+Invoke the automated tests:
+
+```bash
+$ py.test -v
+```
+
+You should see all tests PASSED and no errors printed.
+
+# Contributing Guidelines
+
+## For people who wish to write new code
+
+1. Fork the main GitHub repo of Zeus
+2. Read the the CONTRIBUTING.md and Wiki pages to get yourself up to speed
```

Comment:
~~Read the the CONTRIBUTING.md~~ Read **our `[Contributing Docs](https://github.com/alayek/zeus/blob/master/.github/CONTRIBUTING.md)`**.